### ISSUE TITLE
New graphics patch

### ISSUE THREAD

**Issue by enumag** [07th-mod/tatarigoroshi#53]
@ItaloKnox It's even easier this time.

Just download current tata patch into `tatarigoroshi-old`.

There are some missing images in this repository, add those as well.

Download shell script here: https://www.dropbox.com/s/clcwp0q6f2acnwh/tatarigoroshi.sh?dl=0

Run the script, post errors here.

Add textboxes: https://github.com/07th-mod/resources/releases/download/Nipah/Higurashi-Textboxes.zip

Delete compiled CGAlt, upload the rest.

**Comment by ItaloKnox** [comment]
Got a lot of errors this time. Not sure if I did something wrong.

```

cp: cannot stat 'tatarigoroshi-old/CG/black_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/07thlogo_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/mangagamer_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/logomask_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/white_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/2_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/scenario_a_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/5_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/left_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/right_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/up_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/down_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/c_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/end_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/x_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/cinema_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/m1_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/aa_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata001_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/mask_1900_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/oka1_Shinken.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/tomita1_Shinken.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/tomita1_Ikari_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/tomita1_Def.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/oka1_Def_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/tomita1_Odoroki.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/oka1_Odoroki_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/tomita1_Warai_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/oka1_Warai.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/tomita1_Kanashimi.png': No such file or directory
cp: cannot stat 'spectrum/ps3/s19/04/500400395.txt': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata002_j.png': No such file or directory
cp: cannot stat 'spectrum/ps3/s19/00/502600001.txt': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/bg_235_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/kameda2a_Shinken_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/kameda2a_Warai_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/kameda2a_Def_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/kameda2a_Niyari_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata003_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata004_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata005_j.png': No such file or directory
cp: cannot stat 'spectrum/ps3/s03/02/140200066_1.txt': No such file or directory
cp: cannot stat 'spectrum/ps3/s03/02/140200066_2.txt': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata008_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata009_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata010_02_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata010_04_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/toketu1c_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/red_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/toketu1a_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/toketu1b_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma01_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma02_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma03_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma04_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma05_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma06_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_kuruma07_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata011_03_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_jisitu01_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_jisitu02_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata_jisitu03_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CGAlt/oka1_Tere_0.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata012_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/tata013_02_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/s_jump_back_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/4_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_otsu_tata_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/3_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/centerblind_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/e1.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/e1_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/end_1.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/end_1_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/end_3.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/end_3_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_jump.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_jump_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_otsu.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_otsu_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_otsu_wata.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_otsu_wata_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_tips.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/ex_tips_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/haikei-.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/haikei-_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/haikei_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/mask1013.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/mask1013_j.png': No such file or directory
cp: cannot stat 'tatarigoroshi-old/CG/no_data_j.png': No such file or directory
```

**Comment by enumag** [comment]
These are all ok actually. It's caused by tata having lot of text images.

**Comment by ItaloKnox** [comment]
[Tatarigoroshi-Voices.zip](https://github.com/07th-mod/resources/releases/download/Hanyuu/Tatarigoroshi-Voices.zip)
[Tatarigoroshi-CG.7z](https://github.com/07th-mod/resources/releases/download/Hanyuu/Tatarigoroshi-CG.7z)

**Comment by DoctorDiablo** [comment]
I put together a dll with the lipsync changes for Tata.  [Assembly-CSharp_TatariLipsync.zip](https://github.com/07th-mod/tatarigoroshi/files/1730067/Assembly-CSharp_TatariLipsync.zip)

@EliEron, can you add the changes you made in the Watanagashi dll to this one?  (or tell me what you changed and I can add it)

Looks like we've got the voices and graphics straightened out.  Is the script updated yet?  What else do we need to do?

**Comment by EliEron** [comment]
@DoctorDiablo I only made three changes to the Watanagashi DLL. I fixed the script to operation translation as discussed earlier, I fixed the Mod Menu which was broken and added support for using the numpad when using the number shortcuts.

The script to operation translation and the mod menu seems to be correct in the Tatarigoroshi DLL without me having to change anything. So the only change I needed to make was add the numpad shortcuts, which honestly is not that important a change.

Here is the DLL:
[Tatarigoroshi-Assembly-Numpad.zip](https://github.com/07th-mod/tatarigoroshi/files/1730370/Tatarigoroshi-Assembly-Numpad.zip)
